include/dt-bindings/pinctrl/stm32-pinctrlf1.h

```diff
@@ -75,10 +75,17 @@
  * registers for particular pin.
  */
 
-#define STM32_PIN_USART_TX	(STM32_MODE_OUTPUT | STM32_CNF_ALT_FUNC | STM32_CNF_PUSH_PULL)
-#define STM32_PIN_USART_RX	(STM32_MODE_INPUT | STM32_CNF_IN_FLOAT)
-#define STM32_PIN_I2C		(STM32_MODE_OUTPUT | STM32_CNF_ALT_FUNC | STM32_CNF_OPEN_DRAIN)
-#define STM32_PIN_PWM		(STM32_MODE_OUTPUT | STM32_CNF_ALT_FUNC | STM32_CNF_PUSH_PULL)
-
+#define STM32_PIN_USART_TX		(STM32_MODE_OUTPUT | STM32_CNF_ALT_FUNC | STM32_CNF_PUSH_PULL)
+#define STM32_PIN_USART_RX		(STM32_MODE_INPUT | STM32_CNF_IN_FLOAT)
+#define STM32_PIN_I2C			(STM32_MODE_OUTPUT | STM32_CNF_ALT_FUNC | STM32_CNF_OPEN_DRAIN)
+#define STM32_PIN_PWM			(STM32_MODE_OUTPUT | STM32_CNF_ALT_FUNC | STM32_CNF_PUSH_PULL)
+#define STM32_PIN_SPI_MASTER_NSS	(STM32_MODE_INPUT | STM32_CNF_IN_FLOAT)
```

Comment:
Shouldn't master NSS be output push pull?

Reply:
According to RM0008 NSS can be configured as
```
Hardware master /slave -> Input floating/ Input pull-up / Input pull-down
Hardware master/ NSS output enabled -> Alternate function push-pull
Software -> Not used. Can be used as a GPIO
```
I just added the first option to support multi master bus.

Reply:
Since this part is a bit tricky, it might be worse adding some comments.
You could also provide definitions for NSS output enabled, even if not used.
It might help user to understand what he need.
Exact reference to User manual (§25.3.1: Slave select (NSS) pin management) would also help.